### PR TITLE
chore: make comment and implementation of `move` consistent

### PR DESCRIPTION
**Pull request by zhiqiangxu**

### CODE REVIEW

File: packages/contracts-bedrock/src/dispute/lib/LibPosition.sol [hunk 1]
```diff
@@ -166,8 +166,8 @@ library LibPosition {
     }
 
     /// @notice Get the move position of `_position`, which is the left child of:
-    ///         1. `_position + 1` if `_isAttack` is true.
-    ///         1. `_position` if `_isAttack` is false.
+    ///         1. `_position` if `_isAttack` is true.
+    ///         2. `_position | 1` if `_isAttack` is false.
```
Comment (trianglesphere) [inline packages/contracts-bedrock/src/dispute/lib/LibPosition.sol:170]: @clabby for review

**Comment by refcell** [comment]
Note @zhiqiangxu, specs moved to a new repository so we should update this there instead if possible. A pr is on the merge queue right now to remove the specs in this monorepo.

New repo: https://github.com/ethereum-optimism/specs

**Review by clabby — APPROVED** [review]
LGTM; Thanks! Just need to rebase on `develop` to resolve conflicts once https://github.com/ethereum-optimism/optimism/pull/9121 is merged - specs no longer live in the monorepo as @refcell noted.

**Comment by zhiqiangxu** [comment]
I've removed the spec change since it's already moved here: https://github.com/ethereum-optimism/specs/pull/3

**Review by refcell — APPROVED** [review]

**Pull request merged**